installing_on_kubernetes/_topics/external_postgres.md

```diff
@@ -0,0 +1,14 @@
+## External postgresql
+
+Running with an external Postgres server is an option, if you want the default internal Postgres you can skip this step. Additionally, if you want to secure the connection, you need to include the optional parameters `sslmode=verify-full` and `rootcertificate` when creating the secret. To do this, manually create the secret substituting your values before creating the CR.
+
+        $ oc create secret generic postgresql-secrets \
+          --from-literal=dbname=vmdb_production \
+          --from-literal=hostname=YOUR_HOSTNAME \
+          --from-literal=port=5432 \
+          --from-literal=password=YOUR_PASSWORD_HERE \
+          --from-literal=username=YOUR_USERNAME_HERE \
+          --from-literal=sslmode=verify-full \ # optional
+          --from-file=rootcertificate=path/to/optional/certificate.pem # optional
+
+Note: If desired, the secret name is also customizable by setting `databaseSecret` in the CR.
```

Comment:
Avoid "desired" and use "wanted or needed".
Note: If wanted, the secret name is also customizable by setting `databaseSecret` in the CR.

Reply:
Done